packages/ui/src/components/PasswordStrengthMeter/__stories__/index.stories.tsx

```diff
@@ -9,6 +9,11 @@ export default {
         component: 'Show strength of a password based on different criteria.',
       },
     },
+    parameters: {
```

Comment:
There is already parameters at higher level, this way won't work we don't see the component deprecated on storybook